manhattan_generator.py

```diff
@@ -636,7 +645,7 @@ def parse_args():
                                            comma
     ``--significant-threshold``   Float    The significant threshold for
                                            linkage
-    ``--no-annotation``           Boolean  Do not draw annotation (SNP names)
+    ``--no_annotation``           Boolean  Do not draw annotation (SNP names)
```

Comment:
Please stay with previous options, so that it doesn't change existing scripts (`--no-annotation`).